airflow/providers/apache/sqoop/CHANGELOG.rst

```diff
@@ -28,6 +28,17 @@
 Changelog
 ---------
 
+4.0.0
```

Comment:
please add 4.0.0 entry to provider.yaml

Reply:
sorry, missed adding this earlier. Pushed a commit now to reflect this in the provider.yaml